### PR TITLE
#RI-2513, #RI-2514, #RI-2515, #RI-2516, #RI-2517, #RI-2521

### PR DESCRIPTION
**Pull request by zalenskiSofteq**
#RI-2513 - Edit mode doesn't have modules list
#RI-2514 - Number of icons should be depend on screen resolution
#RI-2515 - Hovering over each module icon, tooltip hides and appears again
#RI-2516 - On Browser page module icon should display tooltip with info per 1 module
#RI-2517 - +N number should be displayed only when N>=2
#RI-2521 - When user adds new DB with port not 6379 but with relevant list of modules, DB is highlighted

### CODE REVIEW

File: redisinsight/ui/src/components/database-list-modules/DatabaseListModules.tsx [hunk 1]
```diff
@@ -138,42 +139,70 @@ const DatabaseListModules = React.memo(({ modules, inCircle, highlight, tooltipT
...
+        ? Module(moduleName, abbreviation, icon, content)
+        : (
+          <EuiToolTip
+            position="bottom"
+            display="inlineBlock"
+            content={Content[i]}
+            anchorClassName={styles.anchorModuleTooltip}
+            key={moduleName}
+          >
+            <>
```
Comment (AmirAllayarovSofteq) [inline redisinsight/ui/src/components/database-list-modules/DatabaseListModules.tsx:177]: May better remove unnecessary <></>?

**Review by AmirAllayarovSofteq — DISMISSED** [review]
LGTM!

File: redisinsight/ui/src/components/database-list-modules/DatabaseListModules.tsx (hunk 1 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/components/database-list-modules/DatabaseListModules.tsx:177]: removed

**Review by AmirAllayarovSofteq — APPROVED** [review]
LGTM

File: redisinsight/ui/src/components/database-list-modules/DatabaseListModules.tsx (hunk 1 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/components/database-list-modules/DatabaseListModules.tsx:177]: update:
without with Fragment unit tests are failed

**Review by AmirAllayarovSofteq — APPROVED** [review]
LGTM

**Pull request merged**